activate.sh

```diff
@@ -153,6 +153,12 @@ autoenv_cd() {
 }
 
 # Override the cd alias
+if setopt 2> /dev/null | grep -q aliasfuncdef; then
+	alias_func_def_status=true;
```

Comment:
I think this `alias_func_def_status` should be `has_alias_func_def_enabled` for more meaningful.